index.html

```diff
@@ -332,23 +437,193 @@ <h4>SvgRenderingTemplate</h4>
       "name": "Bachelor of Science and Arts"
     }
   },
-  <span class="highlight">"renderMethod": [{
-    "id": "https://example.edu/credentials/BachelorDegree.svg",
-    "type": "SvgRenderingTemplate",
-    "name": "Portrait Mode",
-    "css3MediaQuery": "@media (orientation: portrait)",
-    "digestMultibase": "zQmAPdhyxzznFCwYxAp2dRerWC85Wg6wFl9G270iEu5h6JqW"
-  }]
+  <span class="highlight">"renderMethod": {
+    "type": "TemplateRenderMethod",
+    "renderSuite": "svg-mustache",
```

Comment:
```suggestion
    "renderSuite": "mustache",
```

It may be more efficient to make this change and exclusively on `renderMethod.template.mediaType` for the media type. I am recommending this because there would be no need to update the spec to add a new `*-mustache` `renderSuite`, so long as the range for `mediaType` remains as the standard MIME range.

If we accept this change, than `template` would always need to be an object to encompass `mediaType` and perhaps inline data could be passed to `renderMethod.template.data`.

Thinking out loud - happy to discuss.

Reply:
There seem to be three possible things at play here (regardless of where they're expressed):
- the template engine (i.e. [Mustache](https://mustache.github.io/)) - currently expressed _as part of_ `renderSuite`
- the input format - currently most meaningfully expressed in `template.mediaType`
- the output format - currently not expressed anywhere...

Mustache (as a template engine) can handle any text format. The result (from Mustache's perspective) is just more text. It doesn't care about the input format (as long as it's given text and some data), and it's up to other software what to do with the output text.

The value of `template.mediaType` is used (presumably) to content negotiate on the `template.id` URL value and potentially throw an error if a `text/html` page comes back instead (making it similar to `digestMultibase`).

There is currently no expression of desired output format--which may be fine as that may be a concern for the wallet depending on it's modality and accessibility requirements (i.e. use the SVG to create a PNG or PDF, or read the text of the SVG, etc.). So, that may not need to be expressed by the issuer at all.

So, given those three things, I'm not sure `"renderSuite": "svg-mustache"` adds any value unless the intent is to also convey the issuer's request for an output format.

Consequently, I currently agree with @kezike and think we can drop `svg-` from this particular `renderSuite` value.

Reply:
@BigBlueHat,

If `svg-` is removed from the render suite, it seems to me that wallets will be forced to fetch externalized render methods to determine whether or not they can support them (as opposed to just ignoring them). For this render method:

```
"renderMethod": {
  "id": "https://degrees.example/bachelors-template",
  "type": "TemplateRenderMethod",
  "renderSuite": "svg-mustache",
  "digestMultibase": "zQmG270iEu5h6JqWAPdhyxz2dRerWC85Wg6wFl9znFCwYxAp"
}
```

If `svg-` removed from the render suite, and the template found when resolving the above `id` uses a format that the wallet does not support, then the request will have been made unnecessarily. Another way around this is to say that the `mustache` render suite has a very specific (constrained) set of media types it utilizes so that implementers know that it can't be anything else. But, that leads me to believe a future render suite with `something-else-mustache` will just have to be created to support other types. It's also not clear to me what the baseline requirements (e.g., how many media types need to be supported) would be on implementers.

Reply:
I don't see consensus in the discussion above, so am going to just leave things as-is and hope that one of you raises an issue to either remove the media type (needed for conneg), or remove the "svg" in the renderSuite name (needed if media type isn't specified).